### DISCUSSION TITLE
I am trying to install omd in SUSE 15 SLES 5

### DISCUSSION THREAD

**Discussion by aindrila028** [ConSol/omd#187] [Q&A]
I am using command zypper install omd

I am getting error 

53 new packages to install.
Overall download size: 465.5 MiB. Already cached: 0 B. After the operation, additional 2.1 GiB will be used.
Continue? [y/n/v/...? shows all options] (y): y
Retrieving: libodbc2-2.3.9-150400.16.3.3.x86_64 (sle-module-basesystem)                                                                 (1/53), 173.3 KiB
**Failed to mount cd:/?devices=/dev/disk/by-id/ata-VMware_Virtual_SATA_CDRW_Drive_00000000000000000001 on : Mounting media failed**

**Comment by sni** [comment]
moved to discussions, doesn't look like an omd problem.

**Comment by SubhodeepBandyopadhyay** [comment]
Following
